src/modules/interfaces.ts

```diff
@@ -1,4 +1,4 @@
-import { readdirSync, } from 'fs'
+import fs from 'fs'
```

Comment:
out of curiosity, any reason for this change?

Reply:
Easier to grep for `fs\.` since that module is inconsistent on method capitalization.

But the change also happened because I thought I could remove it but in the end couldn't.